src/bindings/c/src/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ set(TARGET_NAME openvino_c)
 
 # Suppress warnings due to catch macro with legacy exception types
 ov_deprecated_no_errors()
```

Comment:
should we remove ov_deprecated_no_errors since we put OPENVINO_SUPPRESS_DEPRECATED_START into the code?

Reply:
Removed OPENVINO_SUPPRESS_DEPRECATED_START/END in the code